### DISCUSSION TITLE
Meeting of Remix CC (2023-10-30)

### DISCUSSION THREAD

**Discussion by drafish** [PlanckerLabs/remix-cc#100] [Meeting of Remix CC]
### 议题
https://forum.plancker.org/t/pip-088-cnx-incubator-2023-q4-proposal/194/3?u=djq

**Comment by drafish** [comment]
### DJQ
1. 与 Remix 讨论捐赠范围，已备注在四季度规划中了 -- https://github.com/PlanckerLabs/remix-cc/discussions/89

**Comment by yiwen4** [comment]
### yiwen
1.完成  remix 文档  审阅

2.与登链开会，登链愿意提供帮助，主要在内容运营，资源共享方面。修改之前的文章、教学视频，发布在登链上。

3.微信公众号运营、Twitter运营
